data/transport missions.txt

```diff
@@ -1679,6 +1676,23 @@ mission "Shady passenger transport 3 - double cross"
 
 
 
+mission "Shady passenger transport 3 - double cross big payment"
+	name "Shady passenger transport big payment"
+	landing
+	invisible
+
+	to offer
+		has "bribed mercenaries"
```

Comment:
This `to offer` doesn't guarantee that the player has the required number of credits. The player might have had over 8 mil when they got the mission, but they may have lost enough due to crew salaries in their travels to not have enough when they land.

To help avoid the situation of getting the mission, bribing the pirates, landing without enough credits, then having your money taken away from you later because you now have enough, perhaps the base mission should require more than 8 mil to offer as a cushion. Perhaps 10 mil?

Reply:
Good idea. This will also prevent people from getting completely wiped out if they're unlucky enough to trigger this mission right after they hit 8m.